jormungandr/src/metrics/mod.rs

```diff
@@ -0,0 +1,66 @@
+use crate::blockchain::Ref;
+
+use chain_impl_mockchain::block::Block;
+use jormungandr_lib::time::SecondsSinceUnixEpoch;
+
+use std::sync::Arc;
+
+pub mod backends;
+
+pub trait MetricsBackend {
```

Comment:
Good work on the backends. Though now I'm thinking if maintaining the non-prometheus backend would be too much of a burden.

Reply:
I don't think so, it is rather simple. And I am going to extract the common piece that is responsible for new block processing to make it even easier.

Anyway, right now we have a tendency for removing metrics, not adding them. :)